index.js

```diff
@@ -1,40 +1,35 @@
-// This is the Web Server
 const express = require('express');
-// bring in jwt
 const jwt = require('jsonwebtoken');
-// bring in secret
 require('dotenv').config();
 const { JWT_SECRET = "notsosecret" } = process.env;
 const { getUserById } = require("./db")
 
 const server = express();
 
-// create logs for everything
 const morgan = require('morgan');
 server.use(morgan('dev'));
 
-// handle application/json requests
 server.use(express.json());
 
-// here's our static files
 const path = require('path');
 server.use(express.static(path.join(__dirname, 'build')));
 
-// Authorization Middleware to jwt.verify and set req.user
+server.disable('etag')
```

Comment:
this is good for testing in dev, but we should remove or comment this from our repo code.